.gitignore

```diff
@@ -1,6 +1,8 @@
+.DS_Store
 *.idea
 
 .gradle
+build/
```

Comment:
I added the same gitignore line but like: `/build/` which I think you might need if a build dir shows up lower down the line? 

I looked up some examples and they had this:
```# Build output directies
/target
*/target
/build
*/build```